docs/core/preview3/tools/dotnet-clean.md

```diff
@@ -0,0 +1,63 @@
+---
+title: dotnet-clean command | Microsoft Docs
+description: The dotnet-clean command cleans the current directory.
+keywords: dotnet-clean, CLI, CLI command, .NET Core
+author: blackdwarf
+ms.author: mairaw
+ms.date: 10/12/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: eff65fa1-bab4-4421-8260-d0a284b690b2
+---
+
+#dotnet-clean (Tooling )
+
+> [!WARNING]
+> This topic applies to Visual Studio 2017 RC - .NET Core Tools TODO:addtoolingstring 
+
+## Name 
+dotnet-clean -- Cleans the output of a project 
+
+## Synopsis
+
+`dotnet clean [--help] [--output]  [--framework]  
+    [--configuration]  [--runtime] [--verbosity]
+    [<project>]`
+
+## Description
+The `dotnet clean` command will clean the output of the previous build. It is implemented as an MSBuild target, so the project will get evaluated. Only the outputs that were created during the build will be cleaned. Both intermediate (`obj`) and final output (`bin`) folders are cleaned. 
+
+## Options
+
+`-h|--help`
+
+Prints out a short help for the command.  
+
+`-o|--output <OUTPUT_DIRECTORY>`
+
+Directory in which the built binaries were placed. You also need to define `--framework` when you specify this option. If you didn't specify this property during build time, you don't have to specify it for clean.
+
+`-f|--framework <FRAMEWORK>`
+
+The framework that was specified for the build that is cleaned. If you didn't specify this property during build time, you don't have to specify it for clean. The framework needs to be defined in the [project file](csproj.md).
```

Comment:
The framework that was specified for the build that is cleaned -> this reads a bit strange